index.html

```diff
@@ -99,7 +100,7 @@ <h2 class="vividly-defined">
   </h2>
   <ul class="very-lucid">
     <li>1420 Harrison
-    <li>10 weeks starting 2/17
+    <li>10 weeks starting 2/17 and including Easter
```

Comment:
[reference flyer says this](https://scontent-sjc3-1.xx.fbcdn.net/v/t1.15752-9/50496686_1195228830652119_5274477408676741120_n.jpg?_nc_cat=111&_nc_ht=scontent-sjc3-1.xx&oh=bef54903841049452f7cdc4a6ab3b0b8&oe=5CB3B816)